packages/nextjs/services/database/seed.ts

```diff
@@ -7,15 +7,39 @@ import { Client } from "pg";
 
 dotenv.config({ path: path.resolve(__dirname, "../../.env.development") });
 
+const VERCEL_DB_STRING = "verceldb";
+const MAX_SUBMISSIONS_BEFORE_FORCE = 20;
+
+const client = new Client({
+  connectionString: process.env.POSTGRES_URL,
+});
+
 // TODO: protect, only for dev.
 async function seed() {
-  const client = new Client({
-    connectionString: process.env.POSTGRES_URL,
-  });
+  if (process.env.POSTGRES_URL?.includes(VERCEL_DB_STRING)) {
+    console.error("Cannot seed production database");
+    process.exit(1);
+  }
```

Comment:
What if we do this the opposite way?

Checking if `POSTGRES_URL` has the string `localhost`. If not, we assume it's a live database.

Reply:
> Checking if POSTGRES_URL has the string localhost. If not, we assume it's a live database.

Nice didn't notice we had `locahost` in dev URl. This seems alot better to me! updated at https://github.com/BuidlGuidl/extensions-hackathon/pull/59/commits/527c2adc4eff2b2450049e639b513bf3f65e63f5

Reply:
yep, I agree! Let's remove the 20 lenght check.